src/libs/blueprint/conduit_blueprint_mesh_examples.cpp

```diff
@@ -2193,11 +2192,11 @@ basic(const std::string &mesh_type,
 
     // TODO(JRC): Consider removing this code if the extra complexity of having
     // the "offsets" array in the basic examples is decided to be a non-issue.
-    Node &topo = res["topologies"].child(0);
-    if(topo.has_child("elements") && topo["elements"].has_child("offsets"))
-    {
-        topo["elements"].remove("offsets");
-    }
+    // Node &topo = res["topologies"].child(0);
+    // if(topo.has_child("elements") && topo["elements"].has_child("offsets"))
+    // {
+    //     topo["elements"].remove("offsets");
+    // }
```

Comment:
If this code is no longer necessary, the code and its associated comment should be removed.